pkg/cluster/inventory.go

```diff
@@ -275,8 +299,8 @@ func (i *DefaultInventory) MarkForDeletion(runtimeID string) (*State, error) {
 func (i *DefaultInventory) Delete(runtimeID string) error {
 	dbOps := func(tx *db.TxConnection) error {
 		newClusterName := fmt.Sprintf("deleted_%d_%s", time.Now().Unix(), runtimeID)
+		// TODO: Rewrite with grom to stay consistend
```

Comment:
I think we should rewrite the entire inventory if we touch it already, otherwise im afraid we could into too many inconsistencies. Buuut if it keeps the PR small then we can do a follow-up (please open an issue for that then so the team can take it).

Either way small typo :D 
```suggestion
		// TODO: Rewrite with gorm to stay consistend
```